src/docs/resources/platform-adaptations.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Platform Specific Behaviors and Adaptations
```

Comment:
=> Platform-specific behaviors and adaptations

Reply:
Changed here and everywhere below to capitalize first word only.